scripts/ci/jenkins_diff.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/sh -le
+
+# This is an Jenkins-specific script that runs infracost on the current branch.
+# Usage docs: https://www.infracost.io/docs/integrations/
+
+fix_env_vars () {
+    # Jenkins has problems with envs case sensivity
+    iac_path=${iac_path:-$IAC_PATH}
+    terraform_plan_flags=${terraform_plan_flags:-$TERRAFORM_PLAN_FLAGS}
+    terraform_workspace=${terraform_workspace:-$TERRAFORM_WORKSPACE}
+    usage_file=${usage_file:-$USAGE_FILE}
+    config_file=${config_file:-$CONFIG_FILE}
+    fail_condition=${fail_condition:-$FAIL_CONDITION}
+}
+
+process_args () {
+  # Set variables based on the order for GitHub Actions, or the env value for other CIs
+  iac_path=${1:-$iac_path}
+  terraform_plan_flags=${2:-$terraform_plan_flags}
+  terraform_workspace=${3:-$terraform_workspace}
+  usage_file=${4:-$usage_file}
+  config_file=${5:-$config_file}
+  fail_condition=${7:-$fail_condition}
```

Comment:
I think this can be deleted since env vars are used to run the script (not params passed into the script like GH Action).

Reply:
Sure.